octobot_trading/exchanges/config/ccxt_exchange_settings.py

```diff
@@ -0,0 +1,258 @@
+import octobot_trading.exchanges.parser as parser
+
+
+class CCXTExchangeTestStatus:
+    """
+    pass either is_fully_tested=True or
```

Comment:
Good idea

Reply:
we can also use the info later for the webinterface and octobot.online to display detailed exchange support status